src/kernel/heap.zig

```diff
@@ -1,17 +1,23 @@
 const std = @import("std");
-const mem = std.mem;
+const builtin = @import("builtin");
+const build_options = @import("build_options");
+const mock_path = build_options.mock_path;
+const is_test = builtin.is_test;
 const testing = std.testing;
 const Allocator = std.mem.Allocator;
 const Bitmap = @import("bitmap.zig").Bitmap(usize);
+const vmm = if (is_test) @import(mock_path ++ "vmm_mock.zig") else @import("vmm.zig");
+const log = @import("log.zig");
+const panic = @import("panic.zig").panic;
 
-pub const Error = error{ /// A value provided isn't a power of two
-NonPowerOfTwo, /// A pointer being freed hasn't been allocated or has already been freed
+const Error = error{ /// A value provided isn't a power of two
+NotPowerOfTwo, /// A pointer being freed hasn't been allocated or has already been freed
```

Comment:
Wasn't it if you add the comma at the end it fixes it?

Reply:
Yep, looks like I didn't push the fix :)